From e46d749532d1ca5f74d5c4d8d690972675e5c638 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@theqtcompany.com>
Date: Wed, 10 Feb 2016 09:02:09 +0200
Subject: [PATCH] Allow a tools-only build

---
 qt3d.pro | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/qt3d.pro b/qt3d.pro
index a26e76d..016fb30 100644
--- a/qt3d.pro
+++ b/qt3d.pro
@@ -1,4 +1,4 @@
-requires(contains(QT_CONFIG, opengl))
+!tools-only:requires(contains(QT_CONFIG, opengl))
 
 load(configure)
 qtCompileTest(assimp)
@@ -17,3 +17,8 @@ load(qt_parts)
 
 OTHER_FILES += \
     sync.profile
+
+tools-only {
+    sub_tools.depends -= sub_src
+    SUBDIRS = sub_tools
+}
