Index: git/src/qml/qml.pro
===================================================================
--- git.orig/src/qml/qml.pro
+++ git/src/qml/qml.pro
@@ -16,6 +16,11 @@ exists("qqml_enable_gcov") {
     LIBS_PRIVATE += -lgcov
 }
 
+greaterThan(QT_GCC_MAJOR_VERSION, 5) {
+     # Our code is bad. Temporary workaround.
+     QMAKE_CXXFLAGS += -fno-delete-null-pointer-checks -fno-lifetime-dse
+ }
+
 QMAKE_DOCS = $$PWD/doc/qtqml.qdocconf
 
 # 2415: variable "xx" of static storage duration was declared but never referenced
