From c36477f621c7d682ee7140156127ad32e91a40ad Mon Sep 17 00:00:00 2001
From: Paul Eggleton <paul.eggleton@linux.intel.com>
Date: Mon, 29 Oct 2012 20:26:36 -0700
Subject: [PATCH 06/22] Allow building a separate qmake for the target

Upstream-Status: Inappropriate [config]

Signed-off-by: Paul Eggleton <paul.eggleton@linux.intel.com>
Signed-off-by: Mikko Levonmaa <mikko.levonmaa@gmail.com>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 qmake/qmake.pro | 1 +
 1 file changed, 1 insertion(+)

diff --git a/qmake/qmake.pro b/qmake/qmake.pro
index a81d0cd..ed7aebf 100644
--- a/qmake/qmake.pro
+++ b/qmake/qmake.pro
@@ -8,6 +8,7 @@ CONFIG -= qt shared app_bundle uic
 DEFINES += QT_BUILD_QMAKE QT_BOOTSTRAPPED \
     PROEVALUATOR_FULL PROEVALUATOR_DEBUG
 DESTDIR = ../bin/
+TARGET = qmake
 
 OBJECTS_DIR = .
 MOC_DIR = .
-- 
1.8.3.1

