qt5nmapper:  Change property string to onTextString 

Upstream-Status: Pending
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>

diff --git a/content_org/Switch.qml b/content/Switch.qml
index 08a2cba..a8b8b26 100644
--- a/content_org/Switch.qml
+++ b/content/Switch.qml
@@ -5,7 +5,7 @@ Item {
 
     property alias text: textItem.text
     property bool checked: false
-    property string onText: "On"
+    property string onTextString: "On"
     property string offText: "Off"
 
     QtObject {
@@ -76,7 +76,7 @@ Item {
             color: "#000000"
             font.pixelSize: 18
             font.bold: true
-            text: onText
+            text: onTextString
         }
         Text {
             anchors.verticalCenter: parent.verticalCenter
