From c44af9a6f0eff02b439ef770755da551f281e9ad Mon Sep 17 00:00:00 2001
From: rathmann <rathmann@29e17f8d-cf37-471e-a288-72cc03a76eda>
Date: Sun, 29 Mar 2020 09:11:29 +0000
Subject: [PATCH] missing qpainterpath.h include added needed with Qt 5.15
 Beta2

Partial backport from
https://sourceforge.net/p/qwt/code/3187/

---
diff -uNr qwt-6.1.4.orig/src/qwt_compass_rose.cpp qwt-6.1.4/src/qwt_compass_rose.cpp
--- qwt-6.1.4.orig/src/qwt_compass_rose.cpp	2019-01-02 17:21:31.280460826 +0100
+++ qwt-6.1.4/src/qwt_compass_rose.cpp	2020-04-14 13:33:27.214932546 +0200
@@ -11,6 +11,7 @@
 #include "qwt_point_polar.h"
 #include "qwt_painter.h"
 #include <qpainter.h>
+#include <qpainterpath.h>
 
 static QPointF qwtIntersection(
     QPointF p11, QPointF p12, QPointF p21, QPointF p22 )
diff -uNr qwt-6.1.4.orig/src/qwt_dial_needle.cpp qwt-6.1.4/src/qwt_dial_needle.cpp
--- qwt-6.1.4.orig/src/qwt_dial_needle.cpp	2019-01-02 17:21:31.284460922 +0100
+++ qwt-6.1.4/src/qwt_dial_needle.cpp	2020-04-14 13:33:37.006864250 +0200
@@ -13,6 +13,7 @@
 #include "qwt_painter.h"
 #include <qapplication.h>
 #include <qpainter.h>
+#include <qpainterpath.h>
 
 #if QT_VERSION < 0x040601
 #define qFastSin(x) qSin(x)
diff -uNr qwt-6.1.4.orig/src/qwt_null_paintdevice.cpp qwt-6.1.4/src/qwt_null_paintdevice.cpp
--- qwt-6.1.4.orig/src/qwt_null_paintdevice.cpp	2019-01-02 17:21:31.296461212 +0100
+++ qwt-6.1.4/src/qwt_null_paintdevice.cpp	2020-04-14 13:31:21.983807348 +0200
@@ -9,6 +9,7 @@
 
 #include "qwt_null_paintdevice.h"
 #include <qpaintengine.h>
+#include <qpainterpath.h>
 #include <qpixmap.h>
 
 class QwtNullPaintDevice::PrivateData
diff -uNr qwt-6.1.4.orig/src/qwt_painter_command.h qwt-6.1.4/src/qwt_painter_command.h
--- qwt-6.1.4.orig/src/qwt_painter_command.h	2020-04-14 13:29:35.128556140 +0200
+++ qwt-6.1.4/src/qwt_painter_command.h	2020-04-14 13:28:23.957056341 +0200
@@ -15,6 +15,7 @@
 #include <qpixmap.h>
 #include <qimage.h>
 #include <qpolygon.h>
+#include <qpainterpath.h>
 
 class QPainterPath;
 
diff -uNr qwt-6.1.4.orig/src/qwt_painter.cpp qwt-6.1.4/src/qwt_painter.cpp
--- qwt-6.1.4.orig/src/qwt_painter.cpp	2019-01-02 17:21:31.452464985 +0100
+++ qwt-6.1.4/src/qwt_painter.cpp	2020-04-14 13:30:22.356224890 +0200
@@ -19,6 +19,7 @@
 #include <qpainter.h>
 #include <qpalette.h>
 #include <qpaintdevice.h>
+#include <qpainterpath.h>
 #include <qpixmap.h>
 #include <qstyle.h>
 #include <qtextdocument.h>
diff -uNr qwt-6.1.4.orig/src/qwt_plot_glcanvas.cpp qwt-6.1.4/src/qwt_plot_glcanvas.cpp
--- qwt-6.1.4.orig/src/qwt_plot_glcanvas.cpp	2019-01-02 17:21:31.300461309 +0100
+++ qwt-6.1.4/src/qwt_plot_glcanvas.cpp	2020-04-14 13:33:14.947018129 +0200
@@ -12,6 +12,7 @@
 #include "qwt_painter.h"
 #include <qevent.h>
 #include <qpainter.h>
+#include <qpainterpath.h>
 #include <qdrawutil.h>
 #include <qstyle.h>
 #include <qstyleoption.h>
diff -uNr qwt-6.1.4.orig/src/qwt_plot_panner.cpp qwt-6.1.4/src/qwt_plot_panner.cpp
--- qwt-6.1.4.orig/src/qwt_plot_panner.cpp	2019-01-02 17:21:31.468465371 +0100
+++ qwt-6.1.4/src/qwt_plot_panner.cpp	2020-04-14 13:32:33.259309129 +0200
@@ -14,6 +14,7 @@
 #include <qbitmap.h>
 #include <qstyle.h>
 #include <qstyleoption.h>
+#include <qpainterpath.h>
 
 #if QT_VERSION >= 0x050000
 #if QT_VERSION < 0x050100
diff -uNr qwt-6.1.4.orig/src/qwt_plot_renderer.cpp qwt-6.1.4/src/qwt_plot_renderer.cpp
--- qwt-6.1.4.orig/src/qwt_plot_renderer.cpp	2019-01-02 17:21:31.352462567 +0100
+++ qwt-6.1.4/src/qwt_plot_renderer.cpp	2020-04-14 13:31:51.019604276 +0200
@@ -20,6 +20,7 @@
 #include "qwt_math.h"
 
 #include <qpainter.h>
+#include <qpainterpath.h>
 #include <qtransform.h>
 #include <qprinter.h>
 #include <qfiledialog.h>
diff -uNr qwt-6.1.4.orig/src/qwt_widget_overlay.cpp qwt-6.1.4/src/qwt_widget_overlay.cpp
--- qwt-6.1.4.orig/src/qwt_widget_overlay.cpp	2019-01-02 17:21:31.388463437 +0100
+++ qwt-6.1.4/src/qwt_widget_overlay.cpp	2020-04-14 13:31:07.091911561 +0200
@@ -11,6 +11,7 @@
 #include "qwt_painter.h"
 #include <qpainter.h>
 #include <qpaintengine.h>
+#include <qpainterpath.h>
 #include <qimage.h>
 #include <qevent.h>
 
diff -uNr qwt-6.1.4.orig/examples/dials/attitude_indicator.cpp qwt-6.1.4/examples/dials/attitude_indicator.cpp
--- qwt-6.1.4.orig/examples/dials/attitude_indicator.cpp	2019-01-02 17:21:31.164458020 +0100
+++ qwt-6.1.4/examples/dials/attitude_indicator.cpp	2020-04-14 13:44:46.354218570 +0200
@@ -3,6 +3,7 @@
 #include <qwt_round_scale_draw.h>
 #include <qevent.h>
 #include <qpainter.h>
+#include <qpainterpath.h>
 #include <qpolygon.h>
 
 AttitudeIndicatorNeedle::AttitudeIndicatorNeedle( const QColor &color )
diff -uNr qwt-6.1.4.orig/src/qwt_plot_glcanvas.h qwt-6.1.4/src/qwt_plot_glcanvas.h
--- qwt-6.1.4.orig/src/qwt_plot_glcanvas.h	2020-04-14 13:44:26.642354964 +0200
+++ qwt-6.1.4/src/qwt_plot_glcanvas.h	2020-04-14 13:42:38.555103164 +0200
@@ -12,6 +12,7 @@
 
 #include "qwt_global.h"
 #include <qframe.h>
+#include <qpainterpath.h>
 #include <qgl.h>
 
 class QwtPlot;
