From 40e6c0f88774b55111c63ed28bba626141797476 Mon Sep 17 00:00:00 2001
From: Mikko Levonmaa <mikko.levonmaa@palm.com>
Date: Tue, 4 Dec 2012 15:32:33 -0800
Subject: [PATCH 3/3] qtbase: Allow qt.conf override when bootstrapping

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Mikko Levonmaa <mikko.levonmaa@palm.com>
---
 qmake/option.cpp |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/qmake/option.cpp b/qmake/option.cpp
index 91c154a..a4f3a85 100644
--- a/qmake/option.cpp
+++ b/qmake/option.cpp
@@ -647,6 +647,10 @@ qmakeAddCacheClear(qmakeCacheClearFunc func, void **data)
 
 QString qt_libraryInfoFile()
 {
+    QString qtconfig = QFile::decodeName(getenv("QT_CONF_PATH"));
+    if (QFile::exists(qtconfig)) {
+        return qtconfig;
+    }
     if (!Option::globals->qmake_abslocation.isEmpty())
         return QDir(QFileInfo(Option::globals->qmake_abslocation).absolutePath()).filePath("qt.conf");
     return QString();
-- 
1.7.4.1

