From 838bb0afe93321ae3d225988728a5736cd4844fc Mon Sep 17 00:00:00 2001
From: Cleiton Bueno <cleitonrbueno@gmail.com>
Date: Thu, 24 Dec 2015 15:59:51 -0200
Subject: [PATCH] Include dependency to QCoreApplication::translate()

Signed-off-by: Cleiton Bueno <cleitonrbueno@gmail.com>
---
 src/core/media_capture_devices_dispatcher.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/core/media_capture_devices_dispatcher.h b/src/core/media_capture_devices_dispatcher.h
index 579d159..0904c46 100644
--- a/src/core/media_capture_devices_dispatcher.h
+++ b/src/core/media_capture_devices_dispatcher.h
@@ -45,6 +45,8 @@
 #include <list>
 #include <map>
 
+#include <QCoreApplication>
+
 #include "web_contents_adapter_client.h"
 
 #include "base/callback.h"
