From 70d76cb38e611c42411d623f165ec27eaa4ba615 Mon Sep 17 00:00:00 2001
From: Frieder Schrempf <frieder.schrempf@online.de>
Date: Mon, 1 Dec 2014 14:34:40 +0000
Subject: [PATCH] functions.prf: allow build for linux-oe-g++ platform

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Frieder Schrempf <frieder.schrempf@online.de>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 tools/qmake/mkspecs/features/functions.prf | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tools/qmake/mkspecs/features/functions.prf b/tools/qmake/mkspecs/features/functions.prf
index 1287799..fb06b21 100644
--- a/tools/qmake/mkspecs/features/functions.prf
+++ b/tools/qmake/mkspecs/features/functions.prf
@@ -5,6 +5,12 @@ defineTest(isPlatformSupported) {
       return(false)
     }
     gcc:!clang:!isGCCVersionSupported(): return(false)
+  } else:linux-oe-g++* {
+    !gcc:!clang {
+      skipBuild("Qt WebEngine on Linux requires clang or GCC.")
+      return(false)
+    }
+    gcc:!clang:!isGCCVersionSupported(): return(false)
   } else:win32 {
     winrt {
       skipBuild("WinRT is not supported.")
