From a1c248259f3d8561d6e2c75a56edf4a72c393843 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Thu, 15 Aug 2019 07:52:53 +0300
Subject: [PATCH] musl: allow build without glibc

Signed-off-by: Samuli Piippo <samuli.piippo@qt.io>
---
 mkspecs/features/configure.prf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkspecs/features/configure.prf b/mkspecs/features/configure.prf
index cc84182b..f1310f75 100644
--- a/mkspecs/features/configure.prf
+++ b/mkspecs/features/configure.prf
@@ -75,7 +75,7 @@ defineTest(runConfigure) {
             return(false)
         }
 
-        !qtConfig(webengine-system-glibc) {
+        false:!qtConfig(webengine-system-glibc) {
             skipBuild("A suitable version >= 2.27 of libc could not be found.")
             return(false)
         }
