From dd72b83e84d0c9f8034096ff2db3d826ba708eb6 Mon Sep 17 00:00:00 2001
From: Cleiton Bueno <cleitonrbueno@gmail.com>
Date: Fri, 25 Dec 2015 18:16:05 -0200
Subject: [PATCH] WebEngine qquickwebengineview_p_p.h add include QColor

Signed-off-by: Cleiton Bueno <cleitonrbueno@gmail.com>
---
 src/webengine/api/qquickwebengineview_p_p.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/webengine/api/qquickwebengineview_p_p.h b/src/webengine/api/qquickwebengineview_p_p.h
index 0fd187a..54b4b7f 100644
--- a/src/webengine/api/qquickwebengineview_p_p.h
+++ b/src/webengine/api/qquickwebengineview_p_p.h
@@ -56,6 +56,8 @@
 #include <QString>
 #include <QtCore/qcompilerdetection.h>
 #include <QtGui/qaccessibleobject.h>
+#include <QColor>
+
 
 namespace QtWebEngineCore {
 class WebContentsAdapter;
