diff -u -r1.1 tools/qmake/mkspecs/features/configure.prf
--- a/tools/qmake/mkspecs/features/configure.prf  2015/02/04 10:24:55     1.1
+++ b/tools/qmake/mkspecs/features/configure.prf  2015/02/04 10:26:52
@@ -20,7 +20,7 @@
         qtCompileTest($$test)
     }
     # libcap-dev package doesn't ship .pc files on Ubuntu.
-    linux:!config_libcap:skipBuild("libcap appears to be missing")
+    #linux:!config_libcap:skipBuild("libcap appears to be missing")
     contains(QT_CONFIG, xcb) {
         for(package, $$list("libdrm xcomposite xi xrandr")) {
             !packagesExist($$package):skipBuild("Unmet dependency: $$package")

