From a5c8d833ffb8b54626b7b670b2782a6cf9a45075 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 12 Nov 2019 19:53:59 -0800
Subject: [PATCH] mkspecs: Allow builds with libc != glibc

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 mkspecs/features/configure.prf | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/mkspecs/features/configure.prf b/mkspecs/features/configure.prf
index cc84182b..c278ffae 100644
--- a/mkspecs/features/configure.prf
+++ b/mkspecs/features/configure.prf
@@ -75,11 +75,6 @@ defineTest(runConfigure) {
             return(false)
         }
 
-        !qtConfig(webengine-system-glibc) {
-            skipBuild("A suitable version >= 2.27 of libc could not be found.")
-            return(false)
-        }
-
         QT_FOR_CONFIG += gui-private
 
         !qtConfig(webengine-system-khr) {
-- 
2.24.0

