From 6ed60f35761ab268004727be544e9ccf70594211 Mon Sep 17 00:00:00 2001
From: Cleiton Bueno <cleitonrbueno@gmail.com>
Date: Fri, 25 Dec 2015 18:16:05 -0200
Subject: [PATCH] WebEngine qquickwebengineview_p_p.h add include QColor

Signed-off-by: Cleiton Bueno <cleitonrbueno@gmail.com>
---
 src/webengine/api/qquickwebengineview_p_p.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/webengine/api/qquickwebengineview_p_p.h b/src/webengine/api/qquickwebengineview_p_p.h
index f13bfd0..0e12244 100644
--- a/src/webengine/api/qquickwebengineview_p_p.h
+++ b/src/webengine/api/qquickwebengineview_p_p.h
@@ -60,6 +60,8 @@
 #include <QString>
 #include <QtCore/qcompilerdetection.h>
 #include <QtGui/qaccessibleobject.h>
+#include <QColor>
+
 
 namespace QtWebEngineCore {
 class WebContentsAdapter;
