From 3102dad3fd5887c1c0a2eea2466d985d6279683f Mon Sep 17 00:00:00 2001
From: Cleiton Bueno <cleitonrbueno@gmail.com>
Date: Thu, 24 Dec 2015 15:59:51 -0200
Subject: [PATCH] Include dependency to QCoreApplication::translate()

Signed-off-by: Cleiton Bueno <cleitonrbueno@gmail.com>
---
 src/core/media_capture_devices_dispatcher.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/core/media_capture_devices_dispatcher.h b/src/core/media_capture_devices_dispatcher.h
index b13a9da..8a54cce 100644
--- a/src/core/media_capture_devices_dispatcher.h
+++ b/src/core/media_capture_devices_dispatcher.h
@@ -46,6 +46,8 @@
 #include <map>
 #include <QtCore/qcompilerdetection.h>
 
+#include <QCoreApplication>
+
 #include "web_contents_adapter_client.h"
 
 #include "base/callback.h"
