From a8163b060b699aa6678c45c970f4fbc806a6c198 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 7 Jul 2017 17:41:43 -0700
Subject: [PATCH] chromium: musl: link with libexecinfo on musl

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 chromium/base/base.gyp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chromium/base/base.gyp b/chromium/base/base.gyp
index 3c3aded640..347b7e7856 100644
--- a/chromium/base/base.gyp
+++ b/chromium/base/base.gyp
@@ -126,6 +126,7 @@
               '-lrt',
               # For 'native_library_linux.cc'
               '-ldl',
+              '-lexecinfo',
             ],
           },
           'conditions': [
