From 9563ef306503f2503a09aadce63020b3048a185d Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Wed, 19 Mar 2014 18:32:28 +0100
Subject: [PATCH 07/11] qeglplatformintegration: Undefine CursorShape from X.h

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/platformsupport/eglconvenience/qeglplatformintegration_p.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/platformsupport/eglconvenience/qeglplatformintegration_p.h b/src/platformsupport/eglconvenience/qeglplatformintegration_p.h
index 4d7adce..c9b6873 100644
--- a/src/platformsupport/eglconvenience/qeglplatformintegration_p.h
+++ b/src/platformsupport/eglconvenience/qeglplatformintegration_p.h
@@ -50,6 +50,12 @@
 #include <QtCore/QVariant>
 #include <EGL/egl.h>
 
+// Undefine CursorShape from X.h, which is causing breakage in Qt::CursorShape in platformsupport/eglconvenience/
+// /usr/include/X11/X.h:#define CursorShape           0       /* largest size that can be displayed */
+#ifdef CursorShape
+#undef CursorShape
+#endif
+
 QT_BEGIN_NAMESPACE
 
 class QEGLPlatformScreen;
-- 
2.1.1

