From 4699b5b9af628e120db9ab26e25bc7069d35562a Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Wed, 19 Mar 2014 18:32:28 +0100
Subject: [PATCH 06/12] qeglplatformintegration: Undefine CursorShape from X.h

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/platformsupport/eglconvenience/qeglplatformintegration_p.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/platformsupport/eglconvenience/qeglplatformintegration_p.h b/src/platformsupport/eglconvenience/qeglplatformintegration_p.h
index 2b5d5f5..a0aab03 100644
--- a/src/platformsupport/eglconvenience/qeglplatformintegration_p.h
+++ b/src/platformsupport/eglconvenience/qeglplatformintegration_p.h
@@ -50,6 +50,12 @@
 #include <qpa/qplatformnativeinterface.h>
 #include <EGL/egl.h>
 
+// Undefine CursorShape from X.h, which is causing breakage in Qt::CursorShape in platformsupport/eglconvenience/
+// /usr/include/X11/X.h:#define CursorShape           0       /* largest size that can be displayed */
+#ifdef CursorShape
+#undef CursorShape
+#endif
+
 QT_BEGIN_NAMESPACE
 
 class QEGLPlatformScreen;
-- 
2.4.2

