From 4dd60bf7b9db89fad8154168e71581ed648cf356 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Thu, 22 Jan 2015 16:09:35 +0100
Subject: [PATCH 12/12] Set paths for target properly
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 configure | 43 +++++++++++++++++++++----------------------
 1 file changed, 21 insertions(+), 22 deletions(-)

diff --git a/configure b/configure
index 7a34b11..7bb4476 100755
--- a/configure
+++ b/configure
@@ -3644,28 +3644,27 @@ static const char qt_configure_prefix_path_strs[][512 + 12] = {
     "qt_xmplpath=$QT_SYSROOT_EXAMPLES",
     "qt_tstspath=$QT_SYSROOT_TESTS",
 #else
-    "qt_prfxpath=$QT_INSTALL_PREFIX",
-    "qt_docspath=$QT_INSTALL_DOCS",
-    "qt_hdrspath=$QT_INSTALL_HEADERS",
-    "qt_libspath=$QT_INSTALL_LIBS",
-    "qt_lbexpath=$QT_INSTALL_LIBEXECS",
-    "qt_binspath=$QT_INSTALL_BINS",
-    "qt_plugpath=$QT_INSTALL_PLUGINS",
-    "qt_impspath=$QT_INSTALL_IMPORTS",
-    "qt_qml2path=$QT_INSTALL_QML",
-    "qt_adatpath=$QT_INSTALL_ARCHDATA",
-    "qt_datapath=$QT_INSTALL_DATA",
-    "qt_trnspath=$QT_INSTALL_TRANSLATIONS",
-    "qt_xmplpath=$QT_INSTALL_EXAMPLES",
-    "qt_tstspath=$QT_INSTALL_TESTS",
-    "qt_ssrtpath=$CFG_SYSROOT",
-    "qt_hpfxpath=$QT_HOST_PREFIX",
-    "qt_hbinpath=$QT_HOST_BINS",
-    "qt_hlibpath=$QT_HOST_LIBS",
-    "qt_hdatpath=$QT_HOST_DATA",
-    "qt_ebinpath=$QT_EXTERNAL_HOST_BINS",
-    "qt_targspec=$shortxspec",
-    "qt_hostspec=$shortspec",
+    "qt_prfxpath=$QT_SYSROOT_PREFIX",
+    "qt_docspath=$QT_SYSROOT_DOCS",
+    "qt_hdrspath=$QT_SYSROOT_HEADERS",
+    "qt_libspath=$QT_SYSROOT_LIBS",
+    "qt_lbexpath=$QT_SYSROOT_LIBEXECS",
+    "qt_binspath=$QT_SYSROOT_BINS",
+    "qt_plugpath=$QT_SYSROOT_PLUGINS",
+    "qt_impspath=$QT_SYSROOT_IMPORTS",
+    "qt_qml2path=$QT_SYSROOT_QML",
+    "qt_adatpath=$QT_SYSROOT_ARCHDATA",
+    "qt_datapath=$QT_SYSROOT_DATA",
+    "qt_trnspath=$QT_SYSROOT_TRANSLATIONS",
+    "qt_xmplpath=$QT_SYSROOT_EXAMPLES",
+    "qt_tstspath=$QT_SYSROOT_TESTS",
+    "qt_ssrtpath=",
+    "qt_hpfxpath=$QT_SYSROOT_PREFIX",
+    "qt_hbinpath=$QT_SYSROOT_BINS",
+    "qt_hlibpath=$QT_SYSROOT_LIBS",
+    "qt_hdatpath=$QT_SYSROOT_LIBS/qt5",
+    "qt_targspec=linux-g++",
+    "qt_hostspec=linux-g++",
 #endif
 };
 static const char qt_configure_settings_path_str[256 + 12] = "qt_stngpath=$QT_INSTALL_SETTINGS";
-- 
2.4.2

