From a850a288fb526e6a4003616aa11dd992688ccc89 Mon Sep 17 00:00:00 2001
From: Paul Eggleton <paul.eggleton@linux.intel.com>
Date: Mon, 29 Oct 2012 20:26:36 -0700
Subject: [PATCH 17/17] Allow building a separate qmake for the target

Upstream-Status: Inappropriate [config]

Signed-off-by: Paul Eggleton <paul.eggleton@linux.intel.com>
Signed-off-by: Mikko Levonmaa <mikko.levonmaa@gmail.com>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 qmake/qmake.pro | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/qmake/qmake.pro b/qmake/qmake.pro
index 89d6ea5..7c605b9 100644
--- a/qmake/qmake.pro
+++ b/qmake/qmake.pro
@@ -8,6 +8,8 @@ CONFIG -= qt
 DEFINES += \
     QT_BUILD_QMAKE \
     PROEVALUATOR_FULL
+DESTDIR = ../bin/
+TARGET = qmake
 
 VPATH += \
     ../src/corelib/global \
-- 
2.0.0

