From 8a2f852417414e19d060da66cd7c5cba76e43c7d Mon Sep 17 00:00:00 2001
From: Diego Rondini <diego.rondini@kynetics.com>
Date: Wed, 19 Jul 2017 14:17:51 +0200
Subject: [PATCH] qtabbar.sip: fix build with accessibility disabled

Upstream-Status: Submitted

Signed-off-by: Diego Rondini <diego.rondini@kynetics.com>
---
 sip/QtWidgets/qtabbar.sip | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/sip/QtWidgets/qtabbar.sip b/sip/QtWidgets/qtabbar.sip
index ffe2d3b..80ad816 100644
--- a/sip/QtWidgets/qtabbar.sip
+++ b/sip/QtWidgets/qtabbar.sip
@@ -166,11 +166,15 @@ protected:
 
 public:
 %If (Qt_5_8_0 -)
+%If (PyQt_Accessibility)
     QString accessibleTabName(int index) const;
 %End
+%End
 %If (Qt_5_8_0 -)
+%If (PyQt_Accessibility)
     void setAccessibleTabName(int index, const QString &name);
 %End
+%End
 
 private:
     QTabBar(const QTabBar &);
-- 
2.7.4

