From 6e25650c829bf1052f607b3f3edbdf88f163efe0 Mon Sep 17 00:00:00 2001
From: Leif Middelschulte <Leif.Middelschulte@klsmartin.com>
Date: Tue, 8 Feb 2022 16:28:14 +0100
Subject: [PATCH] gn.pro: do not try to statically link stdc++

Upstream-Status: Pending
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/buildtools/gn.pro | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/buildtools/gn.pro b/src/buildtools/gn.pro
index a8ca6567b..9632f0ef7 100644
--- a/src/buildtools/gn.pro
+++ b/src/buildtools/gn.pro
@@ -20,7 +20,8 @@ build_pass|!debug_and_release {
 
             gn_gen_args = --no-last-commit-position --out-path $$out_path \
                           --cc \"$$which($$CC_host)\" --cxx \"$$which($$CXX_host)\" \
-                          --ld \"$$which($$CXX_host)\" --ar \"$$which(ar)\"
+                          --ld \"$$which($$CXX_host)\" --ar \"$$which(ar)\" \
+                          --no-static-libstdc++
 
             msvc:!clang_cl: gn_gen_args += --use-lto
 
