From 1a4e33ac630b0e4f285e238479900f0e10c6d15f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Mon, 4 Jun 2018 10:35:46 +0200
Subject: [PATCH] Do not skip build for cross-compile
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 Tools/qmake/mkspecs/features/functions.prf | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Tools/qmake/mkspecs/features/functions.prf b/Tools/qmake/mkspecs/features/functions.prf
index e3f42cec4..797b7085f 100644
--- a/Tools/qmake/mkspecs/features/functions.prf
+++ b/Tools/qmake/mkspecs/features/functions.prf
@@ -53,8 +53,6 @@ defineReplace(appleSdkVersion) {
 defineTest(isPlatformSupported) {
     !qtHaveModule(widgets): skipBuild("QtWidgets module is required to build QtWebKit.")
 
-    cross_compile: skipBuild("cross-compilation of QtWebKit with qmake is not supported yet")
-
     requiredPrograms = cmake gperf python perl bison ruby flex
     for(program, requiredPrograms): \
         !programExistsInPath($$program): \
