From 07ec1f1728be3dff2f93f939065b7fb58f1df434 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Fri, 19 Jul 2013 13:35:14 +0000
Subject: [PATCH] Install the qtwaylandscanner tool to the native side

Upstream-Status: Inappropiate [configuration]

Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 src/qtwaylandscanner/qtwaylandscanner.pro | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/qtwaylandscanner/qtwaylandscanner.pro b/src/qtwaylandscanner/qtwaylandscanner.pro
index ac2d07b..7a46a24 100644
--- a/src/qtwaylandscanner/qtwaylandscanner.pro
+++ b/src/qtwaylandscanner/qtwaylandscanner.pro
@@ -1,6 +1,10 @@
 option(host_build)
 
-SOURCES += qtwaylandscanner.cpp
+TARGET = qtwaylandscanner$$qtPlatformTargetSuffix()
+CONFIG += console warn_off
+QT = core
 
-load(qt_tool)
+SOURCES += qtwaylandscanner.cpp
 
+target.path = $$[QT_HOST_BINS]
+INSTALLS += target
