From 000075567007d0ef76ef3c782fc4060f8ff8e647 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 29 Jan 2019 20:29:19 -0800
Subject: [PATCH] Add missing header for errno

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/bluetooth/bluez/bluetoothmanagement.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/bluetooth/bluez/bluetoothmanagement.cpp b/src/bluetooth/bluez/bluetoothmanagement.cpp
index 31d3dc02..75d762e5 100644
--- a/src/bluetooth/bluez/bluetoothmanagement.cpp
+++ b/src/bluetooth/bluez/bluetoothmanagement.cpp
@@ -45,6 +45,7 @@
 #include "bluez_data_p.h"
 #include "../qbluetoothsocketbase_p.h"
 
+#include <errno.h>
 #include <unistd.h>
 #include <sys/prctl.h>
 #include <sys/syscall.h>
