From 50443809cc261a3f466cdebd3fa2dd4dda6e6a28 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 25 Sep 2018 23:15:08 +0000
Subject: [PATCH] Use correct path prefix

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 share/qtcreator/translations/translations.pro | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/share/qtcreator/translations/translations.pro b/share/qtcreator/translations/translations.pro
index 0df9762fcc..de09bc47ed 100644
--- a/share/qtcreator/translations/translations.pro
+++ b/share/qtcreator/translations/translations.pro
@@ -12,10 +12,10 @@ defineReplace(prependAll) {
     return($$result)
 }
 
-XMLPATTERNS = $$shell_path($$[QT_INSTALL_BINS]/xmlpatterns)
-LUPDATE = $$shell_path($$[QT_INSTALL_BINS]/lupdate) -locations relative -no-ui-lines -no-sort
-LRELEASE = $$shell_path($$[QT_INSTALL_BINS]/lrelease)
-LCONVERT = $$shell_path($$[QT_INSTALL_BINS]/lconvert)
+XMLPATTERNS = $$OE_QMAKE_PATH_EXTERNAL_HOST_BINS/xmlpatterns
+LUPDATE = $$OE_QMAKE_PATH_EXTERNAL_HOST_BINS/lupdate -locations relative -no-ui-lines -no-sort
+LRELEASE = $$OE_QMAKE_PATH_EXTERNAL_HOST_BINS/lrelease
+LCONVERT = $$OE_QMAKE_PATH_EXTERNAL_HOST_BINS/lconvert
 
 wd = $$replace(IDE_SOURCE_TREE, /, $$QMAKE_DIR_SEP)
 
